packages/zoe/test/unitTests/contractSupport/test-zoeHelpers.js

```diff
@@ -369,17 +369,17 @@ test('ZoeHelpers rejectOffer', t => {
       getZoeService: () => {},
       complete: handles => completedOfferHandles.push(...handles),
     });
-    const { rejectOffer } = makeZoeHelpers(mockZoe);
+    const { rejectOffer } = makeZoeHelpers(mockZCF);
     const offerHandles = harden([{}, {}]);
     t.throws(
       () => rejectOffer(offerHandles[0]),
-      /Error: The offer was invalid. Please check your refund./,
+      /The offer was invalid. Please check your refund./,
```

Comment:
This should have been correct as is - was there a reason for this change?

Reply:
It was a mistake due to an intermediate state of changes to the assert package. Reverted.